packages/itwin/grouping-mapping-widget/package.json

```diff
@@ -10,7 +10,8 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/itwin/grouping-mapping-widget"
+    "url": "https://github.com/iTwin/viewer-components-react",
```

Comment:
In the npm documentation, the url is provided with a `.git` suffix. I do not know if it is required, but maybe we would want to add it here as well? `https://github.com/iTwin/viewer-components-react.git` is a valid url.

Reply:
.git is more efficient